File: skiCentar/UI/skicentar_desktop/lib/models/ticket_type.dart

import 'package:json_annotation/json_annotation.dart';
import 'package:skicentar_desktop/models/resort.dart';
import 'package:skicentar_desktop/models/ticket_type_seniority.dart';

part 'ticket_type.g.dart';
@JsonSerializable()
class TicketType{
  int? id;
  bool? fullDay;
  double? price;
  TicketTypeSeniority? ticketTypeSeniority;
  Resort? resort;
  int? ticketTypeSeniorityId;
  int? resortId;

  TicketType({this.id,this.fullDay,this.price,this.ticketTypeSeniorityId,this.resortId});

  factory TicketType.fromJson(Map<String,dynamic> json) => _$TicketTypeFromJson(json);

  Map<String,dynamic> toJson() => _$TicketTypeToJson(this);
}

File: skiCentar/skiCentar.Model/TicketType.cs
namespace skiCentar.Model
{
    public class TicketType
    {
        public int Id { get; set; }

        public int? TicketTypeSeniorityId { get; set; }

        public virtual TicketTypeSeniority? TicketTypeSeniority { get; set; }

        public int? ResortId { get; set; }

        public virtual Resort? Resort { get; set; }

        public bool FullDay { get; set; }

        public decimal Price { get; set; }
    }
}
